ansible/roles/couchdb/tasks/main.yml

```diff
@@ -13,6 +13,15 @@
     - libcurl4-gnutls-dev
     - libtool
 
+- name: Install couchdb pip packages
```

Comment:
You might want to consider using couchdb2

Reply:
Hey @snopoke , even on your current ansible master branch, it looks like you guys use both couchdb and couchdb2. Am I reading this right? Sorry for being thick—just not sure I understand what you mean here.

From line 42 of `deploy_db.yml` on master at commcarehq-ansible:
```yml
- name: Couchdb
  hosts: couchdb
  become: true
  roles:
    - {role: couchdb, tags: 'couchdb'}

- name: Couchdb log rolling configurations
  hosts: couchdb
  become: true
  roles:
    - role: ansible-logrotate
      tags: couchdb
      logrotate_scripts:
        - name: "{{ deploy_env }}_couchdb"
          path: "{{ couch_log_dir }}/*.log"
          options:
            - daily
            - size 100M
            - rotate 10
            - missingok
            - compress
            - delaycompress
            - copytruncate
            - nocreate
            - notifempty

- name: Couchdb 2.0
  hosts: couchdb2
  become: true
  roles:
    - {role: couchdb2, tags: 'couchdb2'}

- name: Couchdb2 log rolling configurations
  hosts: couchdb2
  become: true
  roles:
    - role: ansible-logrotate
      tags: couchdb2
      logrotate_scripts:
        - name: "{{ deploy_env }}_couchdb2"
          path: "{{ couchdb_dir }}/var/log/*.stderr"
          options:
            - daily
            - size 100M
            - rotate 10
            - missingok
            - compress
            - delaycompress
            - copytruncate
            - nocreate
            - notifempty
```